fluentnamer/src/main/java/com/azure/autorest/fluent/model/ResourceTypeName.java

```diff
@@ -19,6 +20,7 @@ public class ResourceTypeName {
     public static final String SUB_RESOURCE_AUTO_GENERATED = SUB_RESOURCE + AUTO_GENERATED;
     public static final String PROXY_RESOURCE_AUTO_GENERATED = PROXY_RESOURCE + AUTO_GENERATED;
     public static final String TRACKED_RESOURCE_AUTO_GENERATED = TRACKED_RESOURCE + AUTO_GENERATED;
+    public static final String EXTENSION_RESOURCE_AUTO_GENERATED = EXTENSION_RESOURCE + AUTO_GENERATED;
```

Comment:
We probably don't need it.

m4 for Swagger do this AutoGenerated. Here I guess we won't see this.

Reply:
Yeah, removed.